openapi.yaml

```diff
@@ -0,0 +1,100 @@
+openapi: 3.0.3
+info:
+  title: Swagger (Task 3 / Product service) - OpenAPI 3.0
+  description: Task 3 (First API with AWS API Gateway and AWS Lambda)
+  termsOfService: http://swagger.io/terms/
+  contact:
+    email: apiteam@swagger.io
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 1.0.11
+externalDocs:
+  description: Find out more about Swagger
+  url: http://swagger.io
+servers:
+  - url: https://5ua826jntg.execute-api.us-west-2.amazonaws.com/prod
+tags:
+  - name: product
+    description: Everything about your Product
+    externalDocs:
+      description: Find out more
+      url: http://swagger.io
+paths:
+  /products:
+    get:
+      tags:
+        - product
+      summary: Add a new product to the store
+      description: Add a new product to the store
```

Comment:
First, no need to write the same in `summary` and `description`. in `description` you can write more details, or skip it entirely.
Second, this is a misleading description.